main.tf

```diff
@@ -456,6 +456,26 @@ resource "null_resource" "confirm_network_healthy" {
   }
 }
 
+##############################################################################
+# OCP Console Patch enablement
+##############################################################################
+resource "null_resource" "ocp_console_management" {
+
+  count      = var.verify_worker_network_readiness ? 1 : 0
+  depends_on = [ibm_container_vpc_cluster.cluster, ibm_container_vpc_cluster.autoscaling_cluster, ibm_container_vpc_worker_pool.pool, ibm_container_vpc_worker_pool.autoscaling_pool]
+  triggers = {
+    enable_ocp_console = var.enable_ocp_console
+  }
+  provisioner "local-exec" {
+    command     = "${path.module}/scripts/ocp_console_patch.sh"
```

Comment:
can you change the name of the script so it doesn't get mixed up with our internal console patch script (which is patching something else). Perhaps call it `enable_disable_ocp_console.sh`

Reply:
Done.